### PR TITLE
Add recent records directive

### PR DESCRIPTION
**Pull request by moradology**
Implementing some ashlar changes. Will finish this up when there's an endpoint for efficiently getting counts... Grabbing the relevant card to try and kill two birds with one stone (ought to mitigate the number of pulls of ashlar people need to make)

screen shot 2015-10-09 at 12 48 41 pm 2

### CODE REVIEW

**Comment by moradology** [comment]
Decided to add the directive to this directive's left on the dashboard while I'm in this part of the code.

screen shot 2015-10-09 at 2 15 53 pm 2

screen shot 2015-10-09 at 2 15 56 pm 2

File: web/app/scripts/recent-counts/recent-counts-controller.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    function RecentCountsController($scope, RecordAggregates) {
+        var ctl = this;
+        init();
+        $scope.$on('driver.state.recordstate:selected', init);
+        return ctl;
+
+        function init() {
+            /* jshint camelcase: false */
+            RecordAggregates.recentCounts().then(function(aggregate) {
+                ctl.record_type_plural = aggregate.plural;
```
Comment (kshepard) [inline web/app/scripts/recent-counts/recent-counts-controller.js]: Seems a little weird that plural is being added to/obtained from the aggregate. Any reason you didn't just grab it from the record type on the client-side?

File: web/app/scripts/recent-proportions/recent-proportions-controller.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    'use strict';
+
+    /* ngInject */
+    function RecentProportionsController() {
+        var ctl = this;
+
+        ctl.proportions = [
+          { proportion: 'About half', label: 'birds' },
+          { proportion: 'Some', label: 'frozen waffles' },
+          { proportion: '1.2%', label: 'covers of \'Freebird\'' }
```
Comment (kshepard) [inline web/app/scripts/recent-proportions/recent-proportions-controller.js]: Cute placeholders, but I think it'd be better to use the old placeholders for demo purposes.

File: web/app/scripts/recent-proportions/recent-proportions.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<div class="tab-content">
+    <div role="tabpanel" class="tab-pane active" id="insight01">
+        <h1><b>{{ recent.proportions[0].proportion }}</b> of all traffic accidents involved <b>{{ recent.proportions[0].label }} last week</b>.</h1>
```
Comment (kshepard) [inline web/app/scripts/recent-proportions/recent-proportions.html]: I realize it's just a placeholder, but might as well replace `traffic accidents` with `{{ ctl.recordType.plural_label | lowercase }}`.

File: web/test/karma.conf.js [hunk 4]
```diff
@@ -73,6 +73,12 @@ module.exports = function(config) {
 
       'app/scripts/navbar/module.js',
       'app/scripts/navbar/**.js',
+      'app/scripts/recent-counts/module.js',
+      'app/scripts/recent-counts/**.js',
+      'app/scripts/recent-proportions/module.js',
+      'app/scripts/recent-proportions/**.js',
+      'app/scripts/elem-stats/module.js',
+      'app/scripts/elem-stats/**.js',
```
Comment (kshepard) [inline web/test/karma.conf.js]: These two lines can be removed.

**Comment by kshepard** [comment]
+1

**Pull request merged**